shared-libs/transitions/test/integration/transitions.js

```diff
@@ -5,19 +5,29 @@ const chaiExclude = require('chai-exclude');
 const db = require('../../src/db');
 const config = require('../../src/config');
 const infodoc = require('@medic/infodoc');
-const updateClinics = require('../../src/transitions/update_clinics');
 
 chai.use(chaiExclude);
 
+let updateClinics;
 let transitions;
 let configGet;
 
 describe('functional transitions', () => {
   beforeEach(() => {
-    configGet = sinon.stub(config, 'get');
+    configGet = sinon.stub();
+    config.init({
+      getAll: sinon.stub().returns({}),
+      get: configGet,
+      getTranslations: sinon.stub()
+    });
+    updateClinics = require('../../src/transitions/update_clinics');
     transitions = require('../../src/transitions/index');
   });
-  afterEach(() => sinon.restore());
+
+  afterEach(() => {
+    sinon.reset();
```

Comment:
So, one of the downsides that I found to the pattern of having to `init` the config/db was that once it is inited with a test object, that implementation will leak to the next test (even if you call `sinon.restore()`).  So saying: 

Calling `sinon.reset()` between the tests was my attempt to mitigate this since at least the behavior of the inited config would not leak across tests.